### DISCUSSION TITLE
ESP32 with 64x64 HUB75E Panel

### DISCUSSION THREAD

**Discussion by Npmr** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#620] [Q&A] [answered]
Hello together :) , 

i have a very small 64x64 panel from Waveshare with a HUB75-E connector with a 1/32 scan. 
Using the documentation of this cool project i got half of it to run. Im using a old ESP32 as a controller. 

I defined the pin usage as follow
`#define R1_PIN 25
#define G1_PIN 26
#define B1_PIN 27
#define R2_PIN 14
#define G2_PIN 12
#define B2_PIN 13
#define A_PIN 23
#define B_PIN 19
#define C_PIN 5
#define D_PIN 17
#define E_PIN 32
#define LAT_PIN 4
#define OE_PIN 15
#define CLK_PIN 16` 

and in the setup method i use
`HUB75_I2S_CFG::i2s_pins _pins={R1_PIN, G1_PIN, B1_PIN, R2_PIN, G2_PIN, B2_PIN, A_PIN, B_PIN, C_PIN, D_PIN, E_PIN, LAT_PIN, OE_PIN, CLK_PIN};
  HUB75_I2S_CFG mxconfig(
    64, // Module width
    64, // Module height
    1, // chain length
    _pins // pin mapping
  );
mxconfig.clkphase = false;
`
to use the defined pins.

Unfortunately only half of the rows showing a sign of life. As you can see in the picture.
Because i could not find a solution for my problem in the other discussions / issues, maybe my search was not that good, i wanted to ask here if you got a idea what the problem is and how to fix it.

The display in question is the following:
https://www.waveshare.com/wiki/RGB-Matrix-P2-64x64 with the back code DCHY-P2-64x64-1515-VP
![IMG_20240418_162303632](https://github.com/mrcodetastic/ESP32-HUB75-MatrixPanel-DMA/assets/20204446/3333455a-52a3-4a6d-90cd-0d5377a9d9bf)

**Comment by mrcodetastic** [comment]
Looks like a B pin issue. Try another GPIO for this.

**Reply by mrcodetastic** [reply]
Check the wiring and wire as well.

**Reply by Npmr** [reply]
Thats it! Thank you for your help and the fast response! 

I will close the discussion :)

**Reply by mrcodetastic** [reply]
No problem. Nothing worse than having an exciting and shiny new led panel to play with that doesn't show stuff properly.

**Comment by davemaster** [comment, marked as answer]
Greetings,

I recommend You, check again those cables, each, continuity.. best regards
